Example/Tests/PinterestFlowLayoutTest.swift

```diff
@@ -0,0 +1,153 @@
+//
+//  PinterestFlowLayoutTest.swift
+//  collection_flow_layout_Example
+//
+//  Created by jowkame on 2/28/18.
+//  Copyright © 2018 CocoaPods. All rights reserved.
+//
+
+import Quick
+import Nimble
+import collection_flow_layout
+
+class PinterestFlowLayoutTest: QuickSpec {
+    private let kScreenWidth: CGFloat = 414
+    private let kScreenHeight: CGFloat = 736
+    
+    override func spec() {
+        let items = ["Twitter", "Facebook", "Instagram", "A", "B", "Frameworks", "Developers", "Source code"]
+        
+        describe("Check tags flow layout") {
```

Comment:
You need to test next:
- column count
- `calculateCollectionViewCellsFrames` method
- delegate